runtime/doc/repeat.txt

```diff
@@ -183,7 +183,10 @@ For writing a Vim script, see chapter 41 of the user manual |usr_41.txt|.
 
 					*:so* *:source* *load-vim-script*
 :[range]so[urce] [file]	Runs |Ex| commands or Lua code (".lua" files) from
```

Comment:
Just this change would be sufficient, imo:

```suggestion
:[range]so[urce] [file]	Runs |Ex| commands or Lua code (".lua" files or filetype=lua buffers) from
```

Reply:
I don't think this is accurate. This PR only changes when behavior when `:source` is called without arguments.